runner/local/simple.go

```diff
@@ -10,10 +10,11 @@ import (
 	"github.com/scootdev/scoot/snapshots"
 )
 
-func NewSimpleRunner(exec execer.Execer, checkouter snapshots.Checkouter) runner.Runner {
+func NewSimpleRunner(exec execer.Execer, checkouter snapshots.Checkouter, saver runner.Saver) runner.Runner {
```

Comment:
Not quite so simple anymore... we can revisit naming later though, when we determine if we need different runner impls.